src/README.md

```diff
@@ -1 +1,3 @@
-# Title
+# Title Web form
+It may be necessary to download flask to run the application. 
+To provide this type "pip install flask" in a prompt line
```

Comment:
Please make sure that there is a newline at the end of the last line as well. See "compare" in PR